submissions/Nemooochka/movie-seats/style.css

```diff
@@ -41,7 +41,7 @@ header {
   -webkit-backface-visibility: hidden;
 }
 .btn:hover, .btn:focus, .btn:active {
-  opacity: 0.7;
+  opacity: 0.6;
```

Comment:
Styles on `:hover` indicate to the user that the cursor is on an interactive element.
Styles on `:active` indicates to the user about the pressed state of the button. 
If styles on both states are equal how users find out about the pressed state of button?

Reply:
Good point, thanks!